android/modules/calendar/src/java/ti/modules/titanium/calendar/CalendarModule.java

```diff
@@ -42,6 +42,13 @@
 	@Kroll.constant public static final int STATE_FIRED = AlertProxy.STATE_FIRED;
 	@Kroll.constant public static final int STATE_SCHEDULED = AlertProxy.STATE_SCHEDULED;
 
+	//region recurrence frequency
+	@Kroll.constant public static final int RECURRENCEFREQUENCY_DAILY = 0;
```

Comment:
Personally, I'd prefer `RECURRENCE_FREQUENCY_DAILY` to `RECURRENCEFREQUENCY_DAILY` (same goes for the rest)

Reply:
This is the way we currently have them for iOS:

Reply:
I think we'd want to "fix" them to match our naming conventions. We'd have to retain the existing definitions and mark them as deprecated for this release, then in 8.0 or later we can remove the ones missing the underscores. So not really a breaking change, but an important deprecation to get in before GA.